services/localega-tsd-proxy/src/main/java/no/elixir/fega/ltp/aspects/PublishMQAspect.java

```diff
@@ -138,11 +138,10 @@ public void publishRemove(Object result) {
   }
 
   private void publishMessage(FileDescriptor fileDescriptor, String type) {
-    String json = gson.toJson(fileDescriptor);
```

Comment:
@yasinmiran Is this the line that does a first "encoding into json" that later was "re-encoded" again inside convertAndSend method, causing the "double encoding/conversion" issue we faced in June/July when debugging LS AAI adaption?

Reply:
Yes, that is correct. Now the encoding happens one time here:
https://github.com/ELIXIR-NO/FEGA-Norway/blob/afdf73d89eca8a2aa867eebbcd1d5e6e5e05bb79/services/localega-tsd-proxy/src/main/java/no/elixir/fega/ltp/config/RabbitMQConfig.java#L86-L89